packages/cryptoplease_ui/lib/src/status_widget.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:flutter/material.dart';
+
+class CpStatusWidget extends StatelessWidget {
+  const CpStatusWidget({
+    Key? key,
+    required this.child,
+    required this.backgroundColor,
```

Comment:
Instead of taking background color directly, let's rather make it a semantical parameter, e.g.:

```dart
enum StatusType { success, info, error }
```

and map it to the color internally.